transcendence-app/src/auth/auth.service.ts

```diff
@@ -62,10 +66,14 @@ export class AuthService {
     const result = salt + '.' + hash.toString('hex');
 
     const { confirmationPassword: _, ...newUser } = user;
-    return this.userService.addUser({
+
+    const avatarDto = await this.localFileService.createRandomSVGFile(12, 512);
+    const userDto = {
       ...newUser,
       avatarId: null,
```

Comment:
I can see, because I'm running it locally, and I'm looking at the database, that at some point the avatarId gets assigned and it's not null anymore. I see that it's happening in userService.AddAvatarAndUser(). This particular program flow you have chosen has the inconvenient drawback of having to assign null in the beginning. What about we refactor this a bit so that we never have to assign null to the avatarId, and like that we can remove `| null` from the avatarId type?

Reply:
(We will also have to manually hardcode some avatars into the mock users we are seeding the database with, but shouldn't be a big problem)
BTW, I don't think this should block the merging of this PR, but still, think about this and let me know your opinion, please.

Reply:
Good point. We plan to make the `avatarId` mandatory in #252